Source/SpatialGDK/Private/SpatialActorChannel.cpp

```diff
@@ -544,7 +544,7 @@ void USpatialActorChannel::OnReserveEntityIdResponse(const worker::ReserveEntity
 		//todo: From now on, this actor channel will be useless. We need better error handling, or a retry mechanism here.
 		return;
 	}
-	UE_LOG(LogSpatialGDKActorChannel, Verbose, TEXT("Received entity id (%d) for: %s."), Op.EntityId.value_or(0), *Actor->GetName());
+	UE_LOG(LogSpatialGDKActorChannel, Warning, TEXT("Received entity id (%d) for: %s."), Op.EntityId.value_or(0), *Actor->GetName());
```

Comment:
Was this deliberated changed? Seems more like Log or Verbose level.

Reply:
Changed back